mangle.json

```diff
@@ -25,6 +25,7 @@
   "props": {
     "cname": 6,
     "props": {
+      "$_vnodeId": "__i",
```

Comment:
A thought: we could use `.__v` since it's not used on options currently, and then the property would nicely match `._original`.